css_tasks/task8/index.html

```diff
@@ -0,0 +1,199 @@
+            />
+          </picture>
+        </span>
+      </div>
+    </section>
+    <!-- About Us -->
+    <section class="section">
+      <div class="section__title-wrapper">
+        <h3 class="section__title-content">About us</h3>
+      </div>
+      <p class="section__text">
+        When you work with us, you are part of a team of committed
+        collaborators. All aspects of our software development process are
+        integrated, iterative and agile.
+      </p>
+      <p class="section__text section__text--primary">
+        Our approach unifies design, development and product management to
+        create exceptional products. We start each project by understanding your
+        business goals, the user’s needs and the product requirements. Then we
+        help translate them into effective and elegant solutions.
+      </p>
+    </section>
+    <div class="jumbotron jumbotron--second">
+      <picture>
+        <source media="(min-width: 992px)" srcset="assets/layer-4@2x-min.jpg" />
+        <source media="(min-width: 576px)" srcset="assets/layer-4-min.jpg" />
+        <img
+          src="assets/layer-4@3x-min.jpg"
+          alt="A person writing on tablet"
+          class="jumbotron__image"
+        />
+      </picture>
+      <section class="section section--services">
+        <div class="section--services-line"></div>
+        <div class="section__title-wrapper section__title-wrapper--services">
+          <h3 class="section__title-content section__title-content--services">
+            Services
+          </h3>
+        </div>
+        <p class="section__text section__text--services">
+          Before we sign a contract or write a line of code our team will spend
+          the necessary time needed to understand your product vision.<br />
+          The outcome of this scoping session is a proposal or high-level
+          statement of work. Based on your requirements, we can provide a
+          complete offering or just the pieces you need.
+        </p>
+      </section>
+    </div>
+    <!-- What we do -->
+    <section class="section section-what-we-do">
+      <div class="section__title-wrapper section__title-wrapper--what-we-do">
+        <h3 class="section__title-content">What we do</h3>
+      </div>
+      <img
+        src="assets/svg/shape-2.svg"
+        class="section__icon"
+        alt="Design icon"
+      />
+      <h4 class="section__header">Design</h4>
+      <p class="section__text section__text--what-we-do">
+        Our design practice offers a full range of services including branding
+        strategy, interaction and visual design and user experience testing.<br />
+        Throughout your project, our designers create and implement visual
+        designs and workflows, solicit user feedback, and work with you to make
+        sure what gets built is what is needed.
+      </p>
+      <img
+        src="assets/svg/shape-3.svg"
+        class="section__icon"
+        alt="Development icon"
```

Comment:
I think it's decorative images so alt should be empty
https://support.siteimprove.com/hc/en-gb/articles/115000013031-Accessibility-Image-Alt-text-best-practices